### PR TITLE
(Tutor home) Implemented Find Student block

### PR DESCRIPTION
**Pull request by TSlashDreamy**
Preview:
firefox_oEE7t7ag1w

Lint: 
Code_t782exkkoL

Do i need to put that block somewhere?

### CODE REVIEW

File: src/assets/img/tutor-home-page/bag.png
Comment (MickhaelSh) [inline src/assets/img/tutor-home-page/bag.png:1]: we already have this icon in project

File: src/constants/translations/en/tutor-home-page.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "findStudentBlock": {
```
Comment (MickhaelSh) [inline src/constants/translations/en/tutor-home-page.json:2]: i don`t see you use all those translations in your component

File: src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+import useBreakpoints from '~/hooks/use-breakpoints'
+import { styles } from '~/containers/tutor-home-page/find-student-block/find-student-block.styles'
+import bag from '~/assets/img/tutor-home-page/bag.png'
+import { translationKey } from '~/containers/tutor-home-page/find-student-block/constants'
+import { authRoutes } from '~/router/constants/authRoutes'
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+import AppButton from '~/components/app-button/AppButton'
+
+const FindStudentBlock = () => {
```
Comment (MickhaelSh) [inline src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx:17]: if this is the similar block to that used on students home page, do it reusable and reuse, no need for duplication

Comment (MickhaelSh) [inline src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx:17]: if this is the similar block to that used on students home page, do it reusable and reuse, no need for duplication

**Review by MickhaelSh — CHANGES_REQUESTED** [review]

File: src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { useCallback, useState } from 'react'
+import { useNavigate } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+
+import TitleBlock from '~/components/title-block/TitleBlock'
+import SearchIcon from '@mui/icons-material/Search'
```
Comment (Tolik170) [inline src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx:7]: organize imports: 
-libs
-services, hooks, containers, components
-utils, const , styles

File: src/constants/translations/en/tutor-home-page.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "popularCategories": {
+      "title": "Popular Categories",
+      "description": "Explore tutoring categories you're passionate about.",
+      "viewMore": "View more"
+    },
+    "howItWorks": {
+      "title": "How it works",
+      "description": "4 steps to easy and funny learning with Space2Study",
+      "selectATutor": {
+        "title": "Select a Tutor",
```
Comment (tsivkadi) [inline src/constants/translations/en/tutor-home-page.json:20]: remove all unnecessary translations. I see that some of them you just copied from the student's page and didn't even change it for the tutor

Comment (TSlashDreamy) [reply src/constants/translations/en/tutor-home-page.json]: > remove all unnecessary translations. I see that some of them you just copied from the student's page and didn't even change it for the tutor

removed it

File: src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx (hunk 3 above)
Comment (TSlashDreamy) [reply src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx]: done it

File: src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx (hunk 2 above)
Comment (TSlashDreamy) [reply src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx]: > if this is the similar block to that used on students home page, do it reusable and reuse, no need for duplication

made one reusable component

File: src/constants/translations/en/tutor-home-page.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
+{
+    "findStudentBlock": {
+      "title": {
+        "student": "A Global Network of Tutors",
```
[the remark this answers is not in the document]
Comment (TSlashDreamy) [reply src/constants/translations/en/tutor-home-page.json]: > students variant on Tutors home page?

removed it

File: src/constants/translations/en/tutor-home-page.json (hunk 1 above)
Comment (TSlashDreamy) [reply src/constants/translations/en/tutor-home-page.json:2]: > i don`t see you use all those translations in your component

yeah, removed it

File: src/assets/img/tutor-home-page/bag.png
Comment (TSlashDreamy) [reply src/assets/img/tutor-home-page/bag.png:1]: > we already have this icon in project

removed the icon

File: src/components/find-block/FindBlock.jsx [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
+import { authRoutes } from '~/router/constants/authRoutes'
+import { styles } from '~/components/find-block/find-block.styles'
+
+const FindBlock = ({ translationKey }) => {
   const [inputValue, setInputValue] = useState('')
   const { t } = useTranslation()
   const navigate = useNavigate()
   const { isMobile } = useBreakpoints()
-  const findOffers = `${authRoutes.findOffers.path}?search=${inputValue}`
+  const path = authRoutes.findOffers.path
```
Comment (MickhaelSh) [inline src/components/find-block/FindBlock.jsx:21]: if u use it only once it doesn`t need variable

**Review by MickhaelSh — APPROVED** [review]

File: src/components/find-block/FindBlock.jsx (hunk 6 above)
Comment (TSlashDreamy) [reply src/components/find-block/FindBlock.jsx]: > if u use it only once it doesn`t need variable

you are right, removed it

**Review by tsivkadi — APPROVED** [review]

**Review by mxrcury — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**